### ISSUE TITLE
Operator 'GeoMatch' for azurerm_web_application_firewall_policy is missing

### ISSUE THREAD

**Issue by MaxiPalle** [terraform-providers/terraform-provider-azurerm#6581]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

Terraform v0.12.24
+ provider.azurerm v2.6.0

### Affected Resource(s)

* `azurerm_web_application_firewall_policy`

### Terraform Configuration Files

```hcl
resource "azurerm_web_application_firewall_policy" "default" {
  name                  = "${var.identifier}-${lower(var.environment)}-gatewaywafdefaultpolicy"
  resource_group_name   = var.rg_name
  location              = var.rg_location

  custom_rules {
    name      = "blockBlacklistedCountries"
    priority  = 1
    rule_type = "MatchRule"

    match_conditions {
      match_variables {
        variable_name = "RemoteAddr"
      }

      operator           = "GeoMatch"
      negation_condition = false
      match_values       = ["AA", "BB", "XY", "YY", "ZZ"]
    }

    action = "Block"
  }
}
```

### Expected behaviour
Custom Application Gateway WAF rule should have been created.

### Actual behaviour
```bash
Error: expected custom_rules.0.match_conditions.0.operator to be one of [IPMatch Equal Contains LessThan GreaterThan LessThanOrEqual GreaterThanOrEqual BeginsWith EndsWith Regex], got GeoMatch

  on main.tf line 156, in resource "azurerm_web_application_firewall_policy" "default":
 156: resource "azurerm_web_application_firewall_policy" "default" {
```

### Steps to Reproduce
1. terraform apply

### References
https://www.terraform.io/docs/providers/azurerm/r/web_application_firewall_policy.html

**Comment by boschcrank** [comment]
Here is a one-line PR to fix the issue: https://github.com/terraform-providers/terraform-provider-azurerm/pull/7181

**Comment by boschcrank** [comment]
I have no idea why this is not closed, but the issue is fixed in v2.14.0.

**Comment by tombuildsstuff** [comment]
Closing since this has been fixed in v2.14 - thanks for the heads up @boschcrank
